build.gradle

```diff
@@ -643,63 +570,66 @@ task optimizePng {
 def MAVEN_PASSWORD_PROPERTY_NAME = 'mekanismMavenPassword'
 def MAVEN_URL_PROPERTY_NAME = 'mekanismMavenUrl'
 
-publishing {
-    publications {
-        maven(MavenPublication) {
-            from components.java
+def mavenPublicationsClosure = {
```

Comment:
Not sure what the point of this is?

Reply:
It mainly was to remove a bunch of IntelliJ's warnings but I think I will revert that as it is the part I am least sure about